Sources/Fluent/Schema/Schema+Field.swift

```diff
@@ -11,14 +11,31 @@ extension Schema {
         public var `default`: Node?
 
         public enum DataType {
-            case id
+            case id(keyType: KeyType?)
             case int
             case string(length: Int?)
             case double
             case bool
             case data
             case custom(type: String)
         }
+        
+        public enum KeyType {
```

Comment:
Out of these options I would find `IdType` the clearest myself as it would be inclusive of other uses of it beyond primary keys (which also is a relational database term, whereas also for example Mongo is now supported). It also differentiates it from `DataType` clearly enough IMO.